packages/maskbook/src/extension/options-page/DashboardComponents/WalletContent.tsx

```diff
@@ -92,30 +94,53 @@ export const WalletContent = forwardRef<HTMLDivElement, WalletContentProps>(func
     const xsMatched = useMatchXS()
     const chainIdValid = useChainIdValid()
 
+    const [collectiblesPage, setCollectiblesPage] = useState(1)
+    const [transactionType, setTransactionType] = useState<FilterTransactionType>(FilterTransactionType.ALL)
+
     const [addToken, , openAddToken] = useModal(DashboardWalletAddERC20TokenDialog)
     const [walletBackup, , openWalletBackup] = useModal(DashboardWalletBackupDialog)
     const [walletDelete, , openWalletDelete] = useModal(DashboardWalletDeleteConfirmDialog)
     const [walletRename, , openWalletRename] = useModal(DashboardWalletRenameDialog)
     const [addAsset, , openAddAsset] = useModal(DashboardWalletAddERC721TokenDialog)
 
-    const {
-        value: collectiblesFromNetwork = [],
-        loading: collectiblesLoading,
-        error: collectiblesError,
-        retry: collectiblesRetry,
-    } = useCollectiblesFromNetwork(account, CollectibleProvider.OPENSEAN)
-    const collectiblesFromDB = useCollectiblesFromDB()
-    const collectibles = [...collectiblesFromNetwork, ...collectiblesFromDB]
+    const { value: collectibles = [], loading: collectiblesLoading, error: collectiblesError } = useCollectibles(
+        account,
+        CollectibleProvider.OPENSEAN,
+        collectiblesPage,
+    )
+
     const [menu, openMenu] = useMenu(
-        <MenuItem onClick={() => openWalletRename({ wallet })}>{t('rename')}</MenuItem>,
+        <MenuItem key="rename" onClick={() => openWalletRename({ wallet })}>
+            {t('rename')}
+        </MenuItem>,
         wallet._private_key_ || wallet.mnemonic.length ? (
-            <MenuItem onClick={() => openWalletBackup({ wallet })}>{t('backup')}</MenuItem>
+            <MenuItem key="backup" onClick={() => openWalletBackup({ wallet })}>
+                {t('backup')}
+            </MenuItem>
         ) : undefined,
-        <MenuItem onClick={() => openWalletDelete({ wallet })} className={color.error} data-testid="delete_button">
+        <MenuItem
+            key="delete"
+            onClick={() => openWalletDelete({ wallet })}
+            className={color.error}
+            data-testid="delete_button">
             {t('delete')}
         </MenuItem>,
     )
 
+    const [transactionTypeMenu, openTransactionTypeMenu] = useMenu(
+        <MenuItem key="all" onClick={() => setTransactionType(FilterTransactionType.ALL)}>
+            {t('all_transactions')} {transactionType === FilterTransactionType.ALL ? <Check fontSize="small" /> : null}
+        </MenuItem>,
+        <MenuItem key="Send" onClick={() => setTransactionType(FilterTransactionType.SEND)}>
```

Comment:
`key="Send"` => `key="Sent"`

Reply:
enum need to be rename?

```suggestion
        <MenuItem key="Sent" onClick={() => setTransactionType(FilterTransactionType.SENT)}>
```